### ISSUE TITLE
#1920674006------------- All page content should be contained by landmarks

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1790]
<strong>PageName:</strong> FeedsPage  

<strong>Source:</strong>`<td role="cell" class="sc-bVVIoq hDUORT" style="box-sizing: border-box; flex: 0 0 auto; width: 140px; text-align: end;"><div sort-value="0.1">0.1 f/s</div></td>`

<strong>Element Location: </strong>td:nth-child(4)

<strong>Impact: </strong>moderate

<strong>Failure Summary:</strong> Fix any of the following:
  Some page content is not contained by landmarks

<strong>Tags:</strong> cat.keyboard,best-practice

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/region?application=axeAPI
<strong>Screenshot:<strong>`C:\Automation\Workspace\Velocity\Screenshot\FeedsPage\1920674006_All page content should be contained by landmarks.png`
